src/OMJulia.jl

```diff
@@ -122,9 +123,12 @@ mutable struct OMCSession
          args4=randstring(10)
       end
       if (Compat.Sys.iswindows())
-         #@assert omc == nothing "A Custom omc path for windows is not supported"
-         if(omc != nothing)
-            open(pipeline(`$omc $args2 $args3$args4`))
+         if (omc != nothing)
+            omhome = replace(dirname(dirname(omc)),r"[/\\]+" => "/")
```

Comment:
With this I can use a custom omc on Windows, but still not two different at once. But this is all I need at the moment.

Reply:
@AnHeuermann Yes this would work for custom omc on Windows, Inorder to use two different versions at once, we need to create the environment variable for omc process and not the julia process, i found a solution for that , i will push it in few minutes